### ISSUE TITLE
Scrollable Table with Fixed Header

### ISSUE THREAD

**Issue by sagarsanjeev** [reactstrap/reactstrap#2377]
Is there a way to create a table within a card that will scroll down in the table without the header instead of overflowing the card. I cant see anything in the docs and the Table component 
Some Thing like this https://v4.mui.com/components/tables/#fixed-header

**Comment by rkoleci** [comment]
yes, you can easily achieve this with position: sticky for the tbl header.

I will try to make a PR this week for this :)

**Comment by rkoleci** [comment]
ReactStrap will not merge my PR as they don't offer such a component at the moment,
but If you wanna do it:

`  <Table {...args}>
      <thead style={{
        background: 'white',
        position: 'sticky',
        top: 0
      }}>`

add the above style to <thead> and will work.

Note: If the page has scroll you might need to handle that as well in relation to the table scroll.
